contributing.md

```diff
@@ -112,6 +112,19 @@ $ANDROID_SDK_ROOT/tools/bin/avdmanager create avd --name "bpk-avd-min" --package
 ```
 </details>
 
+### Detox
+
+> Skip this section if you don't intend to develop React Native components.
+
+Backpack uses [Detox](https://github.com/wix/detox) for e2e testing of React Native components, it requires `applesimutils` to be installed, this can be added by running
+
+```
+brew tap wix/brew
+brew install applesimutils
+```
+
+for any issue with Detox please refer the the [offical documentation](https://github.com/wix/detox/tree/master/docs).
+
```

Comment:
I would put this in the above 'React Native' section as a sub-section.

So you'd have:

```
### React Native

> Skip this section if you don't intend to develop React Native components.

* iOS 
* Android

#### Detox

...etc

```

Reply:
Done 👍